cognite/neat/legacy/rules/exporters/_rules2pydantic_models.py

```diff
@@ -30,8 +29,8 @@
 
     UTC = timezone.utc
 
-EdgeOneToOne: TypeAlias = TypeAliasType("EdgeOneToOne", str)  # type: ignore[valid-type]
-EdgeOneToMany: TypeAlias = TypeAliasType("EdgeOneToMany", list[str])  # type: ignore[valid-type]
+EdgeOneToOne: TypeAlias = str  # type: ignore[valid-type]
+EdgeOneToMany: TypeAlias = list[str]  # type: ignore[valid-type]
```

Comment:
This caused `mypy` to crash for some reason, since this is legacy I don't think it matters much.
![image](https://github.com/user-attachments/assets/2dc09e29-0eda-4c0f-bfdd-e9a2a72aef8d)